src/components/FiltersOrganizations.js

```diff
@@ -185,11 +207,23 @@ const FiltersOrganizations = (props) => {
           value={serviceArea}
         />
 
-        <Text>Last Verified Before:</Text>
+        <Text>Last Verified:</Text>
         <DateFieldPicker selected={lastVerified} onChange={setLastVerified} />
+        <DateFieldPicker
+          selected={lastVerifiedEnd}
+          onChange={setLastVerifiedEnd}
+        />
 
-        <Text>Last Updated Before:</Text>
+        <Text>Last Updated:</Text>
         <DateFieldPicker selected={lastUpdated} onChange={setLastUpdated} />
+        <DateFieldPicker
+          selected={lastUpdatedEnd}
+          onChange={setLastUpdatedEnd}
+        />
+
+        <Text>Created At:</Text>
+        <DateFieldPicker selected={createdAt} onChange={setCreatedAt} />
+        <DateFieldPicker selected={createdAtEnd} onChange={setCreatedAtEnd} />
 
         <Text>Created Before:</Text>
```

Comment:
remember to remove this previous filter.

Reply:
Extraneous date filter removed.